_posts/2015-09-11-daktilo.markdown

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Daktilo
+date: 2015-09-11 00:48:00
+homepage: https://github.com/kronik3r/daktilo
+download: https://github.com/kronik3r/daktilo/archive/master.zip
+demo: http://hitchhiker.ma/daktilo/
+author: Motaquillah Maddane
+thumbnail: daktilo.jpg
+license: MIT License
+---
```

Comment:
If You don't have a license file in Your repository, then please add http://opensource.org/licenses/MIT as `license_link`.